File: app/src/main/java/com/legue/axel/lolsummonertool/network/response/mastery/MasteryResponse.java
package com.legue.axel.lolsummonertool.network.response.mastery;

import com.google.gson.annotations.SerializedName;

import java.util.LinkedHashMap;

public class MasteryResponse {
    public String type;
    public String version;
    @SerializedName("data")
    public LinkedHashMap<String, MasteryDetailResponse> masteryDetailResponse;

    public MasteryResponse() {
    }

    public MasteryResponse(String type, String version, LinkedHashMap<String, MasteryDetailResponse> masteryDetailResponse) {
        this.type = type;
        this.version = version;
        this.masteryDetailResponse = masteryDetailResponse;
    }


}

File: app/src/main/java/com/legue/axel/lolsummonertool/network/response/mastery/MasteryResponse.kt
package com.legue.axel.lolsummonertool.network.response.mastery

import com.google.gson.annotations.SerializedName
import java.util.*

data class MasteryResponse(
        var type: String,
        var version: String,
        @field: SerializedName("data")
        var masteryDetailResponse: LinkedHashMap<String, MasteryDetailResponse>
)
